actions/ah-swagger-plugin.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+const { api, Action } = require('actionhero')
+
+module.exports = class SwaggerAction extends Action {
+  constructor () {
+    super()
+    this.name = 'swaggerJson'
```

Comment:
maybe rename this to just `swagger`?  The action could speak a few different languages one day